src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement.Api.Enduser/Services/ConnectionService.cs

```diff
@@ -265,20 +220,19 @@ private async Task<Result<AssignmentPackage>> AddPackage(Guid fromId, Guid toId,
 
         var assignment = existingAssignments.First();
 
-        var userPackageFilter = ConnectionPackageRepository.CreateFilterBuilder()
-            .Equal(t => t.ToId, DbAudit.Value.ChangedBy)
-            .Equal(t => t.FromId, fromId)
-            .Equal(t => t.PackageId, packageId);
-
-        var userPackags = await ConnectionPackageRepository.GetExtended(userPackageFilter, callerName: SpanName("Get extended packages assignments"), cancellationToken: cancellationToken);
-        var userpackage = userPackags.Where(p => p.PackageId == packageId)
-            .ToList();
-
-        problem = ValidationRules.Validate(
-            ValidationRules.QueryParameters.AnyPackages(userpackage, queryParamName),
-            ValidationRules.QueryParameters.PackageIsAssignableByUser(userpackage, queryParamName),
-            ValidationRules.QueryParameters.PackageIsAssignableByDefinition(userpackage, queryParamName)
-        );
+        var packages = await RelationService.GetPackagePermissionsFromOthers(DbAudit.Value.ChangedBy, fromId, packageId, cancellationToken: cancellationToken);
```

Comment:
Har CanDelegate og IsAssignable. Hvordan sjekke om brukeren kan gi pakken videre?

Reply:
Men må vi egentlig skrive om AddPackage nå? Trodde vi hovedsakelig fokuserte denne omskrivingen for GET endepunktene.